docs/network-flow-visibility.md

```diff
@@ -70,6 +70,50 @@ ConfigMap as shown below. We provide some examples for the parameter values in t
 Please note that the default values for `flowPollInterval` and `flowExportFrequency` parameters are set to 5s and 12, respectively.
 `flowCollectorAddr` is a required parameter that is necessary for the Flow Exporter feature to work.
 
+### IPFIX Fields in a Flow Record
+
+There are 23 IPFIX fields in each flow record. They are from the standard IANA
+ registry, the reverse IANA registry and the Antrea registry for Antrea specific
+  fields. The reverse IANA registry fields are used for flows in the reverse
+   direction. 
```

Comment:
formatting is weird (white spaces at the beginning of the lines)

plus leave empty line before section header